packages/core/src/settings.js

```diff
@@ -8,12 +9,15 @@ import { ENV } from '@pixi/constants';
  * explicitly remove feature support to target a more stable
  * baseline, prefer a lower environment.
  *
+ * Due to {@link https://bugs.chromium.org/p/chromium/issues/detail?id=934823|bug in chromium}
+ * we disable webgl2 by default for all non-apple mobile devices.
+ *
  * @static
  * @name PREFER_ENV
  * @memberof PIXI.settings
  * @type {number}
  * @default PIXI.ENV.WEBGL2
  */
-settings.PREFER_ENV = ENV.WEBGL2;
+settings.PREFER_ENV = !isMobile.any || isMobile.apple.device ? ENV.WEBGL2 : ENV.WEBGL;
```

Comment:
should it be 
`settings.PREFER_ENV = (!isMobile.any || !isMobile.apple.device) ? ENV.WEBGL2 : ENV.WEBGL;` 
would do the trick?

Reply:
I forgot to push. All apple devices are mobile, right?